README.md

```diff
@@ -211,7 +244,7 @@ See [GitHub Actions expressions](https://docs.github.com/en/actions/learn-github
 ### Skip checks on commit messages
```

Comment:
I didn't know what this meant when I first read it, I was like "skip if...there's a commit with only a message? what does that mean?" I think this might be better phrased as "Skip checks if commit message matches a specific pattern" or the like